lib/matplotlib/axes/_subplots.py

```diff
@@ -48,21 +50,27 @@ def __init__(self, fig, *args, **kwargs):
                     raise ValueError(
                         'Single argument to subplot must be a 3-digit '
                         'integer')
-                self._subplotspec = GridSpec(rows, cols)[num - 1]
+                self._subplotspec = GridSpec(rows, cols,
+                                             fig=self.figure)[num - 1]
                 # num - 1 for converting from MATLAB to python indexing
         elif len(args) == 3:
             rows, cols, num = args
             rows = int(rows)
             cols = int(cols)
             if isinstance(num, tuple) and len(num) == 2:
                 num = [int(n) for n in num]
-                self._subplotspec = GridSpec(rows, cols)[num[0] - 1:num[1]]
+                self._subplotspec = GridSpec(
+                        rows, cols,
+                        fig=self.figure)[num[0] - 1:num[1]]
```

Comment:
Feel free to ignore this (especially as you did not really change the formatting), but see https://www.python.org/dev/peps/pep-0008/#pet-peeves (4th point) for spaces around colons.

Reply:
Not ignoring, but just not sure what you mean here!  Is it supposed to be `[num[0] - 1 : num[1]]`?   That would actually make more sense to me.  (To be honest, I didn't even look at this - I just added the `fig=` and made it fit on the line.)